docs/src/pages/customization/palette/palette.md

```diff
@@ -11,6 +11,9 @@ The theme exposes the following color intentions:
 - primary - used to represent primary interface elements for a user.
 - secondary - used to represent secondary interface elements for a user.
 - error - used to represent interface elements that the user should be made aware of.
+- warning - used to represent potentially dangerous actions or important messages.
+- info - used to present information to the user that is neutral and not necessarily important.
```

Comment:
Seem to have lost a review comment from last night: This is different to (and contradicts) the wording at https://github.com/mui-org/material-ui/pull/18820/files/0fc92d5325eb90bcc6799cc34758ac125d0f85a1#diff-9efe743b8cf918063d14bd692661844aR200

@r3dm1ke What was the origin of the proposed wording?

Reply:
My apologies, I missed that. I just committed a change to bring these comments to consistency. I think the definitions in `palette.md` are more accurate, not sure why did I write different ones in `createPalette.js`